### DISCUSSION TITLE
Prebuild failure notification channels (e.g. slack)

### DISCUSSION THREAD

**Discussion by Antman261** [community/community#18974] [Codespaces]
Recently, we've had a couple of issues where we broke prebuilds but didn't realise for a while. Primarily the issue is a matter of notification channel -- **we use slack, and we would like prebuild failure notifications to be sent to a Slack channel.** 

Since we can't add any config to the actions jobs running our prebuilds, this isn't possible right now.

Any chance this capability could be made available?

**Comment by dmgardiner25** [comment]
cc. @sharma-anuradha do we currently have any notifications for prebuild failures other than looking at the actual prebuild itself?
